sdk/ios/Configsum/Inputs/Context.swift

```diff
@@ -53,13 +52,18 @@ public class Context: Codable {
     }
     
     public init(appVersion: String,
-                locale: String,
+                locale: Locale,
                 platform: Platform,
                 osVersion: String,
                 metadata: Metadata?,
-                user: User) {
+                user: User?) {
         let secondsOffset = TimeZone.current.secondsFromGMT()
-        let location = Location(locale: locale, timezoneOffset: secondsOffset)
+        
+        let languageCode = locale.languageCode!
+        let regionCode = locale.regionCode!
+        let formattedLocale = languageCode + "_" + regionCode
```

Comment:
Just a style thing but I think I prefer `"\(languageCode)_\(regionCode)"`. What do you think?